include/uapi/sound/sof/header.h

```diff
@@ -26,6 +26,27 @@ struct sof_abi_hdr {
 	__u32 data[];		/**< Component data - opaque to core */
 }  __packed;
 
+/**
+ * struct sof_ipc4_abi_hdr - Used by any bespoke component data structures or binary blobs.
+ * @magic: The magic number for the header. The value is 'S', 'O', 'F', '4'
+ * @size: The size in bytes of the data, excluding this struct
+ * @abi: SOF ABI version
+ * @blob_type: type of blob: INIT_INSTANCE, CONFIG_SET or LARGE_CONFIG_SET
+ *             The value is of type enum sof_ipc4_module_type
```

Comment:
let's make this clear, not all values of sof_ipc4_module_type can be used, right?

```
 * @blob_type: type of blob: INIT_INSTANCE, CONFIG_SET or LARGE_CONFIG_SET
 *             Three values from enum sof_ipc4_module_type
```

Reply:
I prefer not to add "Three" for later extension.